charts/kube-prometheus-stack/README.md

```diff
@@ -80,9 +82,30 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 45.x to 46.x
+
+This version upgrades Prometheus-Operator to v0.65.1 with new CRDs (PrometheusAgent and ScrapeConfig), Prometheus to v2.43.1 and Thanos to v0.31.0.
+
+As of this release, CRD installation and upgrade is now managed by the chart if the property `crds.enabled` is set to true (by default).
```

Comment:
This is confusing because CRD installation and upgrade is already managed by this chart before the change.